### DISCUSSION TITLE
Feature Request: Enhanced Product Unit Quantity Visibility

### DISCUSSION THREAD

**Discussion by mohammed-fatah** [Blair2004/NexoPOS#2125] [Ideas]
**Description**

In Version: NexoPOS 5.2.6, viewing Product Unit Quantities requires clicking on "Options" for each product. For better visibility and efficiency, I propose adding either:

  An extra column in the main Products List to display Product Unit Quantities directly.
  A new category or section called "Product Unit Quantities List" under Inventory, making it easier to access all product quantities in one place.

**Additional Feature: Stock Color Labeling for Quantities**

To help quickly identify stock levels, it would be useful to implement a color-coded system for the quantity column:

-    Red: Very low stock, based on the threshold set in the "Stock Alert" field.
-    Orange: Warning level, indicating stock is getting low (this could be an additional threshold setting in product listings).
-    Green: Sufficient stock.

This feature would make it much easier to monitor stock levels at a glance, especially for users managing large inventories.

Thank you for considering this enhancement!

**Comment by Blair2004** [comment]
Hi, 
So bad it's just now i noticed this. I'll try to ask the other users about this.

**Reply by mohammed-fatah** [reply]
No worries, I'm sure you are very busy, let me know if you need me to test anything or have any questions. appreciate your time!

**Reply by Blair2004** [reply]
Thanks
